administrator/components/com_workflow/src/Model/WorkflowModel.php

```diff
@@ -297,7 +297,13 @@ public function setDefault($pk, $value = 1)
 		if ($value)
 		{
 			// Unset other default item
-			if ($table->load(array('default' => '1')))
+			if ($table->load(
+				[
+					'default' => '1',
+					'extension' => $table->get('extension')
+				]
+			)
+			)
```

Comment:
```suggestion
				]))
```
I think per coding style, it should be this way?

Reply:
@Quy When I do that I get a notice saying that closing parenthesis should be on a line of their own. What can be done by the looks of it is to put the closing parenthesis together. So I have done that.